Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -412,6 +421,10 @@ void IMixedRealityTouchHandler.OnTouchCompleted(HandTrackingInputEventData event
         {
             if (touchPoints.ContainsKey(eventData.Controller))
             {
+                // Giving a last chance to check if pressed ocurred before another Near interaction becomes the closest to pointer
+                touchPoints[eventData.Controller] = eventData.InputData;
+                UpdateTouch();
+
                 touchPoints.Remove(eventData.Controller);
```

Comment:
Can you update the comment here?

You are update the dictionary's value for key of eventData.controller....but then you remove it from the dictionary. Tracing through the code I see we need it to compute some data. It's just not super obvious.

Reply:
Changed